### DISCUSSION TITLE
Is there a video tutorial on how to create a module?

### DISCUSSION THREAD

**Discussion by giusebos** [joomengine/Joomla-Component-Builder#889] [General]
I would like to know if there is a tutorial on how to make a module, I would like to display a view of my component inside a module.

Thank you

**Comment by Llewellynvdm** [comment]
The long, and best way to learn JCB is not to just pick tutorials you think you need to watch, but to [go over all tutorials](https://www.youtube.com/playlist?list=PLQRGFI8XZ_wtGvPQZWBfDzzlERLQgpMRE).... choose a weekend and just do it, you will not regret this. If your already advanced in PHP, I can understand that is is painfully boring... but there is an option to play at 2x speed :+1: 

There is a [tutorial to build modules](https://www.youtube.com/watch?v=lriWW3Uzyc8&list=PLQRGFI8XZ_wtGvPQZWBfDzzlERLQgpMRE&index=90)... but we don't teach PHP in JCB... this you need to do before you can do advance tasks with PHP, and since the module are in JCB is mostly driven by PHP its considered advance by some.
